File: 1295.Find_Numbers_with_Even_Number_of_Digits.cpp
class Solution
{
public:
    int findNumbers(vector<int> &nums)
    {
        int result = 0;
        for (int i = 0; i < nums.size(); i++)
        {
            int digits = 0;
            while (nums[i] > 0)
            {
                int rem = nums[i] % 10;
                digits++;
                nums[i] = nums[i] / 10;
            }
            if (digits % 2 == 0)
            {
                result++;
            }
        }
        return result;
    }
};

File: 1295.Find_Numbers_with_Even_Number_of_Digits.java
class Solution {
    public static int getDigitCount(int number) {

        int digits = 0;
        while (number > 0) {
            number /= 10;
            digits++;
        }

        return digits;
    }

    public int findNumbers(int[] nums) {
        int ans = 0;
        for (int i = 0; i < nums.length; i++) {
            if (getDigitCount(nums[i]) % 2 == 0) {
                ans++;
            }
        }
        return ans;
    }
}